ee/clickhouse/sql/events.py

```diff
@@ -232,7 +232,7 @@
 """
 
 EVENT_JOIN_PERSON_SQL = """
-INNER JOIN person_distinct_id as pid ON events.distinct_id = pid.distinct_id
+INNER JOIN (SELECT * FROM person_distinct_id WHERE team_id = %(team_id)s) as pid ON events.distinct_id = pid.distinct_id
```

Comment:
It might be every so slightly qucker if you explicitely say `select person_id, distinct_id from`